### PR TITLE
add community and wiki links in navbar

### PR DESCRIPTION
**Pull request by joshinils**
image

TODO:
I would like to have the favicon of the wiki: https://wiki.openstreetmap.org/osm_logo_wiki.png
and the favicon of the community: https://community-cdn.openstreetmap.org/uploads/default/original/1X/e9768dc0a7cbe8d58a55e515a89a6af817534bbf.png 
left of their text, however, hard linking to them where they are now may be a problem.
So, maybe put them on GitHub and link here instead?

### CODE REVIEW

File: osm-org-greasemonkey.user.js [hunk 1]
```diff
@@ -238,6 +238,10 @@ function modifyContent() {
       // KartaView
       thisUrl = "https://kartaview.org/map/@" + OsmMap.lat + "," + OsmMap.lon + "," + OsmMap.zoom + "z";
       createOrUpdate("GM-KARTA", navbar_content, thisUrl, "KartaView", "btn btn-outline-primary");
+      thisUrl = "https://community.openstreetmap.org/";
```
Comment (kmpoppe) [inline osm-org-greasemonkey.user.js:241]:
```suggestion
      // Discourse Community
      thisUrl = "https://community.openstreetmap.org/";
```

File: osm-org-greasemonkey.user.js [hunk 2]
```diff
@@ -238,6 +238,10 @@ function modifyContent() {
...
+      createOrUpdate("GM-COMMU", navbar_content, thisUrl, "<span style=\"color:\">OSM</span> <strong style=\"color:\">Community</strong>", "btn btn-outline-primary");
+      thisUrl = "https://wiki.openstreetmap.org/wiki/Main_Page";
```
Comment (kmpoppe) [inline osm-org-greasemonkey.user.js:245]:
```suggestion
      // Wiki
      thisUrl = "https://wiki.openstreetmap.org/wiki/Main_Page";
```

**Review by kmpoppe — APPROVED** [review]

File: osm-org-greasemonkey.user.js [hunk 3]
```diff
@@ -238,6 +238,13 @@ function modifyContent() {
       // KartaView
       thisUrl = "https://kartaview.org/map/@" + OsmMap.lat + "," + OsmMap.lon + "," + OsmMap.zoom + "z";
       createOrUpdate("GM-KARTA", navbar_content, thisUrl, "<strong style=\"color:#0C1D2E\">Karta</strong><span style=\"color:#635BFF\">View</span>", "btn btn-outline-primary");
+      // Discourse Community
+      // Discourse Community
```
Comment (kmpoppe) [inline osm-org-greasemonkey.user.js]:
```suggestion
```

**Review by kmpoppe — APPROVED** [review]

**Comment by kmpoppe** [comment]
Currently OpenStreetMap.org seems to have a content-policy that prevents loading anything from outside, thus adding the logos next to the text won't work - I like the idea though!

**Comment by joshinils** [comment]
Figured as much.
If they were svg we could insert them as is.
I think technically we could base64 encode the image and insert it in code, that won't load anything from outside, but probably looks bad when reading the code.

Are there any other links we could add?

**Comment by kmpoppe** [comment]
Yeah. Could you add a link to Bing Maps Aerial View at the map location? (the sidebar content was fixed already, remove ref)

**Pull request merged**